src/applications/personalization/dashboard/reducers/folders.js

```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import set from 'lodash/fp/set';
+import set from 'platform/utilities/data/set';
```

Comment:
How did I not know this was a drop-in replacement?! 🤯 

Also, could you prefix all  the new platform imports with `~/`, so `~/platform/utilities/data/set`?

Reply:
Yup! As long as currying isn't involved, this function can be swapped in as a replacement!

And yeah sure, I can add that to the imports. Out of curiosity, is it a matter of consistency/preference? I noticed most of the platform imports in the `personalization` directory have `~/` prepended to them.

Reply:
Yes just a preference and to ensure that no one (like a new dev) thinks that `platform` is from node_modules

Reply:
Yeah that makes sense. The imports should be good now ✅